CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [#41](https://github.com/nf-core/sarek/pull/41) - Update `qualimap` from `2.2.2b` to `2.2.2c`
 - [#41](https://github.com/nf-core/sarek/pull/41), [#55](https://github.com/nf-core/sarek/pull/55) - Update `tiddit` from `2.7.1` to `2.8.1`
 - [#41](https://github.com/nf-core/sarek/pull/41) - Update `vcfanno` from `0.3.1` to `0.3.2`
+- [#46](https://github.com/nf-core/sarek/pull/46) - Add location to abstracts.
```

Comment:
I now realize the issue, #46 and #52 should be in _Added_, not in _Changed_

Reply:
Fixed that :-)